### PR TITLE
site: contest: able to abandon regional event

### PR DESCRIPTION
**Pull request by remi-dupre**
Ables to mark a candidate as "abandoned", thus we can keep track of who participates to a specific event without removing the information that the candidate was qualified on its profile.

### CODE REVIEW

File: prologin/contest/models.py [hunk 1]
```diff
@@ -358,6 +360,10 @@ def is_complete_for_semifinal(self):
     def is_assigned_for_semifinal(self):
         return self.assignation_semifinal == Assignation.assigned.value
 
+    @property
+    def abandoned_for_semifinal(self):
```
Comment (seirl) [inline prologin/contest/models.py]: nit: `has_abandoned_semifinal()`?

**Comment by seirl** [comment]
This looks great. While we're at it, since we're adding this in the Assignation enum, it will also work for the finals. Do you want to take care of the UI side of the finals right now? I'm guessing it should look pretty similar.

**Comment by remi-dupre** [comment]
The thing is that it wouldn't change much thing ?

On the profile, the timeline stops at "selected for the final":


Do we select other people when someone abandons early ?

**Review by zopieux — COMMENTED** [review]
Don't forget to update the French locale.

**Comment by remi-dupre** [comment]
Is there a way to do it before merging the branch?

**Comment by zopieux** [comment]
https://github.com/prologin/site#translations

**Pull request merged**